src/databricks/sql/client.py

```diff
@@ -331,6 +379,10 @@ def execute(
             self.buffer_size_bytes,
             self.arraysize,
         )
+
+        if execute_response.is_staging_operation:
```

Comment:
Question for reviewers: is there any specifically desired end-state for the `cursor` after a staging operation? Maybe we return a new NamedTuple `StagingOperationResult` with properties of `.successful:boolean` and perhaps a copy of the `operation` and `localFile` that were used?

Reply:
I don't quite get this question, but the cursor for now will return just one row and we should have reached the end of this cursor.

Reply:
@susodapop could you please with an a sample code explain how this will provide different experience to the end user?